products/cbbr/sql/spatial_facilities.sql

```diff
@@ -23,8 +23,8 @@ master AS (
     WHERE
         a.geom IS NULL
         AND a.facility_or_park_name IS NOT NULL
-        AND b.facname LIKE '%' || ' ' || '%' || ' ' || '%'
-        AND '%' || UPPER(a.facility_or_park_name) || '%' = '%' || UPPER(b.facname) || '%'
+        AND b.facname LIKE ('%' || ' ' || '%' || ' ' || '%')
```

Comment:
maybe a learning moment for me, but is this different from 
```
AND b.facname LIKE '% % %'
```
(just curious... fine leaving it as it is, since it's not yours)

Reply:
Yeah, fascinating. No it is not